setup/azurecmdfiles.wxi

```diff
@@ -680,6 +680,9 @@
                         <Component Id="cmpC67F14F377421F1BCD74EC5A579D43CB" Guid="*">
                             <File Id="filE9553CEC1D67AB3D4B77759239D60B18" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.IotHub\Microsoft.Azure.Commands.IotHub.dll-Help.xml" />
```

Comment:
Please add these new cmdlet to the changelog for this release.

Reply:
updated